src/kagglehub/exceptions.py

```diff
@@ -58,6 +58,9 @@ def kaggle_api_raise_for_status(response: requests.Response, resource_handle: Op
         response.raise_for_status()
     except requests.HTTPError as e:
         message = str(e)
+        server_error_message = response.json().get("message", "")
```

Comment:
I guess some response are not json format so it fails some integration tests?

Reply:
+1